### DISCUSSION TITLE
Add stuff to left margin

### DISCUSSION THREAD

**Discussion by bsureshkrishna** [quarto-dev/quarto-cli#4087] [Q&A] [answered]
Hi,

Is there a way to add stuff (like a figure or logo) to the left margin in the standard 3-column layout ?

The left margin is currently an empty space since there is no TOC for the landing page.

Thanks, Suresh

**Comment by dragonstyle** [comment, marked as answer]
For websites, when there isn't a sidebar present we typically try to minimize the left margin automatically. You can do a lot, including really taking over the complete layout if you'd like (useful for cases like landing pages). More here:

https://quarto.org/docs/output-formats/page-layout.html#custom
